ocrd_utils/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Pillow >= 6.2.0
 numpy >= 1.17.0
+shapely
```

Comment:
Best to be specific with the versions

Reply:
Mine is currently at `1.6.4.post2`. But I haven't had any problems with Shapely so far, ever.